inst/workflow/1_mappings/STUDY.yaml

```diff
@@ -30,6 +30,24 @@ spec:
     est_fpfv:
       required: true
       type: Date
+    est_lplv:
+      required: true
```

Comment:
should these still be included? If the other PR is removing?

Reply:
depending on the order of merging, i'll take care of this. if it gets merged in before #1936, then we still need the required field